javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,129 @@
+// Doubling of number
+
+const numbers = [1, 2, 3, 4];
+const newNumbers = numbers
+  .filter((x) => x % 2 !== 0)
+  .map((filteredNumber) => filteredNumber * 2);
+console.log(newNumbers);
+
+// Create an array  containing the movies with a short title
+
+const shortMoviesTitle = movies.filter((movie) => !movie.title.includes(" "));
+console.log(shortMoviesTitle);
+
+// Create an array of movie titles with long movie titles
+
+const longMoviesTitles = movies
+  .filter((movie) => movie.title.includes(" "))
+  .map((filteredMovie) => filteredMovie.title);
+console.log(longMoviesTitles);
```

Comment:
Haha that's a pretty crafty idea - short titles are ones with only one word. 

As long as you know that you could filter on e.g. `movies.filter(movie => movie.title.length <= 10)` or something like that, then this is a fine (and sneaky) solution